network/benchmarks/netperf/nptest/nptest.go

```diff
@@ -509,13 +509,18 @@ func (t *NetPerfRPC) ReceiveOutput(data *WorkerOutput, reply *int) error {
 
 func serveRPCRequests(port string) {
 	baseObject := new(NetPerfRPC)
-	rpc.Register(baseObject)
-	rpc.HandleHTTP()
-	listener, e := net.Listen("tcp", ":"+port)
-	if e != nil {
-		log.Fatal("listen error:", e)
+	err := rpc.Register(baseObject)
+	if err == nil {
```

Comment:
if err != nil {
  return
}
// the rest here

[Let's avoid long nesting where possible.]